_includes/checkout-v3-status-models.md

```diff
@@ -815,6 +823,7 @@ responses:
 | {% f externalNonPaymentToken, 2 %} | `string`     | The result of an external tokenization. This value will vary depending on card types, acquirers, customers, etc. For Mass Transit merchants, transactions redeemed by Visa will be populated with PAR. For Mastercard and Amex, it will be our own token. |
 | {% f cardType, 2 %}                | `string`  | `Credit Card` or `Debit Card`. Indicates the type of card used for the authorization.                                                                                                                                                                                                                |
 | {% f maskedPan, 2 %}               | `string`  | The masked PAN number of the card.                                                                                                                                                                                                                                                                   |
+| {% f maskedDPan %}               | `string`  | A masked version of a network token representing the card. It will only appear if the chosen payment instrument is tokenized and the card used is tokenized by Visa or MasterCard.                                                                                                                                                                                                                                                                  |
```

Comment:
**Minor, Nitpick**

`MaskedDPan` has the wrong indentation according to it's place in the model.

![image](https://github.com/SwedbankPay/developer.swedbankpay.com/assets/22122794/59d471d1-9f35-4dc0-a997-c85d875362a6)